pom.xml

```diff
@@ -64,15 +64,16 @@
     <changelist>-SNAPSHOT</changelist>
     <skipIntegrationTests>true</skipIntegrationTests>
     <java.level>8</java.level>
-    <jenkins.version>2.150.3</jenkins.version>
+    <jenkins.version>2.235.1</jenkins.version>
```

Comment:
the current baseline was too old for java 11 support, I selected a more modern one that users will be using based on https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions